config/main.py

```diff
@@ -663,6 +663,17 @@ def _get_disabled_services_list(config_db):
 
     return disabled_services_list
 
+def _stop_all_services():
+    # stop all other services
```

Comment:
And if we were to localize the exception to "Cancellation", we could just try running the "systemctl stop sonic.target" again, since cancellation doesn't mean something is wrong with sonic.target's job.

Reply:
I noticed during my testing that doing a stop on sonic.target doesn't always succeed. The service which caused the conflict needs to be stopped first and then sonic.target needs to be stopped. _stop_all_services() more or less does the same.